label_sync/labels.yaml

```diff
@@ -0,0 +1,355 @@
+#    name: area/example/cassandra
+#  - color: 0052cc
+#    name: area/extensibility
+#  - color: 0052cc
+#    name: area/HA
+#  - color: 0052cc
+#    name: area/hw-accelerators
+#  - color: 0052cc
+#    name: area/images-registry
+#  - color: 0052cc
+#    name: area/ingress
+#  - color: 0052cc
+#    name: area/introspection
+#  - color: 0052cc
+#    name: area/ipv6
+#  - color: 0052cc
+#    name: area/isolation
+#  - color: 0052cc
+#    name: area/kube-proxy
+#  - color: 0052cc
+#    name: area/kubeadm
+#  - color: 0052cc
+#    name: area/kubectl
+#  - color: 0052cc
+#    name: area/kubelet
+#  - color: 0052cc
+#    name: area/kubelet-api
+#  - color: 0052cc
+#    name: area/logging
+#  - color: 0052cc
+#    name: area/monitoring
+#  - color: 0052cc
+#    name: area/node-e2e
+#  - color: 0052cc
+#    name: area/node-lifecycle
+#  - color: 0052cc
+#    name: area/nodecontroller
+#  - color: d4c5f9
+#    name: area/os/coreos
+#  - color: d4c5f9
+#    name: area/os/fedora
+#  - color: d4c5f9
+#    name: area/os/gci
+#  - color: d4c5f9
+#    name: area/os/ubuntu
+#  - color: d4c5f9
+#    name: area/platform/aws
+#  - color: d4c5f9
+#    name: area/platform/azure
+#  - color: d4c5f9
+#    name: area/platform/gce
+#  - color: d4c5f9
+#    name: area/platform/gke
+#  - color: d4c5f9
+#    name: area/platform/mesos
+#  - color: d4c5f9
+#    name: area/platform/vagrant
+#  - color: d4c5f9
+#    name: area/platform/vsphere
+#  - color: 0052cc
+#    name: area/release-infra
+#  - color: 0052cc
+#    name: area/reliability
+#  - color: 0052cc
+#    name: area/rkt
+#  - color: 0052cc
+#    name: area/secret-api
+#  - color: d93f0b
+#    name: area/security
+#  - color: 0052cc
+#    name: area/stateful-apps
+#  - color: 0052cc
+#    name: area/swagger
+#  - color: 0052cc
+#    name: area/system-requirement
+#  - color: 0052cc
+#    name: area/teardown
+#  - color: 0052cc
+#    name: area/test
+#  - color: 0052cc
+#    name: area/test-infra
+#  - color: 0052cc
+#    name: area/third-party-resource
+#  - color: 0052cc
+#    name: area/ui
+#  - color: 0052cc
+#    name: area/upgrade
+#  - color: 0052cc
+#    name: area/usability
+#  - color: 0052cc
+#    name: area/workload-api/cronjob
+#  - color: 0052cc
+#    name: area/workload-api/daemonset
+#  - color: 0052cc
+#    name: area/workload-api/deployment
+#  - color: 0052cc
+#    name: area/workload-api/job
+#  - color: 0052cc
+#    name: area/workload-api/replicaset
+  - color: d93f0b
+    name: beta-blocker
+  - color: fef2c0
+    name: cherrypick-approved
+  - color: fef2c0
+    name: cherrypick-candidate
+  - color: bfe5bf
+    name: 'cla: human-approved'
+  - color: e11d21
+    name: 'cla: no'
+  - color: bfe5bf
+    name: 'cla: yes'
+  - color: e11d21
+    name: 'cncf-cla: no'
+  - color: bfe5bf
+    name: 'cncf-cla: yes'
+  - color: e11d21
+    name: do-not-merge
+  - color: fbca04
+    name: flake-has-meta
+  - color: 006b75
+    name: for-new-contributors
+  - color: 006b75
+    name: help-wanted
```

Comment:
help-wanted can be commented out

(I would eventually like to see this standardized, but my aim for this pass is standard automation labels)